### PR TITLE
Feature/bns 33

### PR DESCRIPTION
**Pull request by davidcormor**
Child issues:

- BNS-36 (Structure Project)
- BNS-37 (User)
- BNS-38 (SDK auth0 - Pending Role)
- BNS-39 (Model Company)
- BNS-79 (Logic Creation Company)
- BNS-136 (Avatar)

### CODE REVIEW

File: src/infrastructure/adapters/auth0/auth0_service.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import json
+from functools import wraps
+
+# Error handler
```
Comment (diegocortes-cafeto) [inline src/infrastructure/adapters/auth0/auth0_service.py:4]: Remove this comment

File: src/infrastructure/adapters/flask/app/controllers/company/company_resources.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+from flask_restx import Resource, Namespace
+from flask_restx.reqparse import request
+
+
+from src.application.company.company_uc import GetCompany, GetAllCompanies, CreateCompany
+from src.domain.entities.common_entity import JwtEntity
+from src.domain.entities.company_entity import CompanyNewEntity
+from src.infrastructure.adapters.auth0.auth0_service import requires_auth
+from src.infrastructure.adapters.flask.app.utils.error_handling import AppErrorBaseClass
+# from app.third_parties.auth0.aurh0_service import requires_auth
```
Comment (diegocortes-cafeto) [inline src/infrastructure/adapters/flask/app/controllers/company/company_resources.py:16]: can we remove these imports?

**Comment by diegocortes-cafeto** [comment]
@davidcormor  in general terms looks good!. Please address the minor code review comments.

File: src/infrastructure/adapters/auth0/auth0_service.py (hunk 1 above)
Comment (davidcormor) [reply src/infrastructure/adapters/auth0/auth0_service.py]: Comment removed

File: src/infrastructure/adapters/database/alembic/README [hunk 3]
```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```
[the remark this answers is not in the document]
Comment (davidcormor) [reply src/infrastructure/adapters/database/alembic/README:1]: Pending update documentation Alembic

File: src/infrastructure/adapters/flask/app/controllers/company/company_resources.py (hunk 2 above)
Comment (davidcormor) [reply src/infrastructure/adapters/flask/app/controllers/company/company_resources.py]: Import removed

**Review by diegocortes-cafeto — APPROVED** [review]
Looks good

**Pull request merged**